test-app/frontend/src/Footer/Footer.scss

```diff
@@ -0,0 +1,40 @@
+@import "~bootstrap/scss/bootstrap";
+
+@font-face {
+    font-family: Sofia Pro Regular;
+    src: url("../../public/fonts/Sofia\ Pro\ Regular\ Az.otf");
+}
+
+@font-face {
+    font-family: Sofia Pro Regular;
+    src: url("../../public/fonts/Sofia\ Pro\ Bold\ Az.otf");
+    font-weight: bold;
+}
+
+:root {
+    /* variables */
+    --cg-black: rgb(30, 32, 32);
+    --cg-blue: rgb(6, 87, 152);
+    --cg-grey: rgb(243, 243, 243);
+    --cg-white: rgb(255, 255, 255);
```

Comment:
It seems unnecessary to have white here. We can just use `white` as the color. No RGB value needed.